src/components/App.js

```diff
@@ -1,17 +1,53 @@
-import React from 'react';
-import Header from './Header';
-import Link from './shared/Link';
-import './App.css';
+import React, { Component, Fragment } from 'react';
+import ErrorBoundary from './shared/ErrorBoundary';
+import { connect } from 'react-redux';
+import IncorrectPath from './IncorrectPath';
+import StartPage from './StartPage';
+import DetailsPage from './DetailsPage';
+import {
+    BrowserRouter as Router, Switch, Route, Redirect
+} from 'react-router-dom';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <Header></Header>
-        <Link></Link>
-      </header>
-    </div>
-  );
+class App extends Component {
+
+    render() {
+        const { error, loading, data } = this.props;
+        return (
+            <Fragment>
+                <Router>
+                    <Switch>
+                        <Route exact path="/">
+                            <ErrorBoundary>
+                                <StartPage data={this.props.data}/>
```

Comment:
I don't have an idea why are you passing the `data` through props, if you connect your component to redux store

Reply:
There is no need to connect Redux as part of this task, so this will be addressed in the future.